### PR TITLE
Throw error if user profile failed to update

### PR DESCRIPTION
**Pull request by Nightrider0098**
## Describe your changes

When profile update api crash. It should throw error with a toast and should not close the modal.

## Issue ticket number and link

https://community.sphinx.chat/bounty/1344

Closes :-  https://github.com/stakwork/sphinx-tribes-frontend/issues/101

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Hi Hitesh. Can you also include in this PR merging the extra test file you created here?
https://github.com/stakwork/sphinx-tribes-frontend/tree/master/src/pages/people

File: src/store/main.ts [hunk 1]
```diff
@@ -1580,6 +1579,15 @@ export class MainStore {
         const [r, error] = await this.doCallToRelay('POST', 'profile', body);
         if (error) throw error;
         if (!r) return;
+        if (!r.ok) {
+          uiStore.setToasts([
+            {
+              id: '1',
```
Comment (elraphty) [inline src/store/main.ts:1585]: This should be in the catch function

Comment (Nightrider0098) [reply src/store/main.ts]: Done

**Comment by ecurrencyhodler** [comment]
@Nightrider0098 can you post a video of the message?

**Pull request merged**